lib/cartopy/mpl/geoaxes.py

```diff
@@ -753,6 +753,25 @@ def add_geometries(self, geoms, crs, **kwargs):
 
         """
         styler = kwargs.pop('styler', None)
+
+        if (not callable(styler)) and ('facecolor' not in kwargs):
+            styler_kw = styler
+
+            def styler(geom):
+                styler_g = styler_kw if styler_kw else dict()
+                styler_g = styler_g.copy()
+
+                if isinstance(geom, sgeom.LineString):
+                    styler_g['facecolor'] = 'none'
+                    if ('edgecolor' not in kwargs) or (
+                            kwargs['edgecolor'] == 'face'):
```

Comment:
```suggestion
                    if kwargs.get('edgecolor', 'face') == 'face':
```
But why would `'face'` mean patch edgecolor?

Reply:
Because `edgecolor = face` means using `facecolor` value for edge. This suggests the user specifies facecolor, and would like to fill the LineString.

Reply:
I'm almost tempted to raise here. Why are we allowing a linestring to be filled at all?